projects/ngx-charts-on-fhir/src/lib/fhir-chart/fhir-chart.component.ts

```diff
@@ -47,6 +50,9 @@ export class FhirChartComponent implements OnInit {
         wheel: { enabled: true },
         drag: { enabled: false },
         pinch: { enabled: true },
+        onZoomComplete: function () {
+          that.isMatGroupFocus = !that.isMatGroupFocus ;
+        }
```

Comment:
why do we need to listen to this event? range-selector is already getting the min/max from `layerManager.timelineRange$` every time the scale bounds change.

Reply:
Also, why is it toggling the state on each zoom? is that really the right behavior?